L79-cpp-byte-buffer-slice-methods.md

```diff
@@ -0,0 +1,53 @@
+C++ API changes on ByteBuffer and Slice
+----
+* Author(s): 
+* Approver: vjpai
+* Status: Draft
+* Implemented in: C++
+* Last updated: 2021-04-19
+* Discussion at: https://groups.google.com/g/grpc-io/c/OsAYW1mDJ9w
+
+## Abstract
+
+For the better integration between gRPC and FlatBuffers, additional methods will be added to `grpc::ByteBuffer` and `grpc::Slice` allowing FlatBuffers to access data without copying data.
+
+## Background
+
+FlatBuffers has been using gRPC core methods to exchange data with gRPC but it's not ideal because gRPC core doesn't promise API stability, which caused it broken time to time.
```

Comment:
caused it broken time to time --> broke it from time to time

Reply:
Done.